images/win/windows2022.json

```diff
@@ -255,7 +255,8 @@
                 "{{ template_dir }}/scripts/Installers/Configure-DynamicPort.ps1",
                 "{{ template_dir }}/scripts/Installers/Configure-GDIProcessHandleQuota.ps1",
                 "{{ template_dir }}/scripts/Installers/Configure-Shell.ps1",
-                "{{ template_dir }}/scripts/Installers/Enable-DeveloperMode.ps1"
+                "{{ template_dir }}/scripts/Installers/Enable-DeveloperMode.ps1",
+                "{{ template_dir }}/scripts/Installers/Windows2022/Install-Win10SDK-17763.ps1"
```

Comment:
let's avoid creating a new file and move the installation to the Install-Vs script as we do for windows-2019
https://github.com/actions/virtual-environments/blob/29907e0bf074d38f5569b40e0b7df6e11b881db3/images/win/scripts/Installers/Install-VS.ps1#L39-L45